extras/cronwrapper

```diff
@@ -15,6 +15,27 @@
 #         (typically /var/log/daemon.log or /var/log/syslog)
 #
 
+sendSlackMessage() {
+    escapedText=$(cat ${1} | sed 's/"/\"/g' | sed "s/'/\'/g" )
+    slackUsername=${SLACK_USERNAME:-plexupdate}
+
+    if [ $RET -eq 10 ] || [ $RET -eq 0 ]; then
+        color="#36a64f"
+    else
+        color="#ad2d2d"
+    fi
+    ts=`date +%s`
```

Comment:
It's [recommended](https://mywiki.wooledge.org/BashFAQ/082) to use the `$()` syntax for command substitution rather than backticks.

Reply:
Sounds good, I'll make the update.